libs/dh/shared/feature-flags/src/lib/dh-feature-flags.ts

```diff
@@ -54,9 +54,13 @@ export const dhFeatureFlagsConfig = {
     created: latestBump,
     disabledEnvironments: [DhAppEnvironment.test_002, DhAppEnvironment.prod],
   },
+  'metering-points-debug': {
```

Comment:
`metering-point-debug` and `metering-points-debug` are kinda similar 😄

Reply:
Yes. For now both are needed :)